app/reporting-framework/json-reports/ovc-patient-list-template.json

```diff
@@ -0,0 +1,181 @@
+        {
+            "type": "simple_column",
+            "alias": "person_id",
+            "column": "t1.person_id"
+        },
+        {
+            "type": "derived_column",
+            "alias": "person_name",
+            "expressionType": "simple_expression",
+            "expressionOptions": {
+                "expression": " CONCAT(COALESCE(person_name.given_name, ''), ' ', COALESCE(person_name.middle_name, ''), ' ', COALESCE(person_name.family_name, ''))"
+            }
+        },
+        {
+            "type": "derived_column",
+            "alias": "identifiers",
+            "expressionType": "simple_expression",
+            "expressionOptions": {
+                "expression": " GROUP_CONCAT(DISTINCT id.identifier SEPARATOR ', ')"
+            }
+        },
+        {
+            "type": "derived_column",
+            "alias": "phone_number",
+            "expressionType": "simple_expression",
+            "expressionOptions": {
+                "expression": " GROUP_CONCAT(DISTINCT contacts.value SEPARATOR ', ')"
+            }
+        },
+        {
+            "type": "simple_column",
+            "alias": "latest_rtc_date",
+            "column": "date_format(fh.rtc_date, '%Y-%m-%d')"
+        },
+        {
+            "type": "simple_column",
+            "alias": "latest_vl",
+            "column": "fh.vl_1"
+        },
+        {
+            "type": "simple_column",
+            "alias": "latest_vl_date",
+            "column": "date_format(fh.vl_1_date, '%Y-%m-%d')"
+        },
+        {
+            "type": "simple_column",
+            "alias": "last_appointment",
+            "column": "CONCAT(COALESCE(DATE_FORMAT(fh.encounter_datetime, '%Y-%m-%d'),''),' ',COALESCE(et.name, ''))"
+        },
+        {
+            "type": "simple_column",
+            "alias": "cur_meds",
+            "column": "fh.cur_arv_meds"
+        },
+        {
+            "type": "simple_column",
+            "alias": "cur_arv_line",
+            "column": "fh.cur_arv_line"
+        },
+        {
+            "type": "simple_column",
+            "alias": "previous_vl",
+            "column": "fh.vl_2"
+        },
+        {
+            "type": "simple_column",
+            "alias": "previous_vl_date",
+            "column": "date_format(fh.vl_2_date, '%Y-%m-%d')"
+        },
+        {
+            "type": "simple_column",
+            "alias": "nearest_center",
+            "column": "pa.address3"
+        },
+        {
+            "type": "simple_column",
+            "alias": "county",
+            "column": "pa.address1"
+        },
+        {
+            "type": "simple_column",
+            "alias": "sub_county",
+            "column": "pa.address2"
+        },
+        {
+            "type": "simple_column",
+            "alias": "ward",
+            "column": "pa.address7"
+        },
+        {
+            "type": "simple_column",
+            "alias": "ovc_identifier",
+            "column": "ovc_id.identifier"
```

Comment:
take county, sub-county, ward and ovc_identifier to come after identifier column